bokeh/crossfilter/models.py

```diff
@@ -282,7 +285,14 @@ def make_plot_choices(self):
           dict: x, y, and plot_type keys and values for initial setup
 
         """
-        x, y = [x['name'] for x in self.continuous_columns()[:2]]
+        cont_columns = self.continuous_columns
+
+        # prefer continuous columns to initialize with, otherwise use what we have
+        if len(cont_columns) > 1:
+            x, y = [x['name'] for x in self.continuous_columns[:2]]
```

Comment:
To be consistent, `cont_columns` or use `self.continuous_columns` for the `len` check